### ISSUE TITLE
IllegalArgumentException when setting snapGravity to center

### ISSUE THREAD

**Issue by clhols** [rubensousa/RecyclerViewSnap#75]
With snap helper version 2.2.1

This is due to missing break; in GravitySnapRecyclerView:

Current:
```
            case 4:
                snapHelper = new GravitySnapHelper(Gravity.CENTER);
            default:
                throw new IllegalArgumentException....
```

should be

```
            case 4:
                snapHelper = new GravitySnapHelper(Gravity.CENTER);
                break;
            default:
                throw new IllegalArgumentException...
```

**Comment by clhols** [comment]
Reverting back to version 2.2.0 from jcenter is a workaround. Seems a regression has happened in 2.2.1.

**Comment by rubensousa** [comment]
Thanks for letting me know. I will fix this in 2.2.2 and will update here once it's released @clhols

**Comment by rubensousa** [comment]
2.2.2 released with this fix
